src/Discord.Net.Core/Entities/Channels/Direction.cs

```diff
@@ -1,14 +1,15 @@
 namespace Discord
 {
     /// <summary>
-     ///     Specifies the direction of where message(s) should be retrieved from.
+    ///     Specifies the direction of where message(s) should be retrieved from.
     /// </summary>
     /// <remarks>
     ///     This enum is used to specify the direction for retrieving messages.
     ///     <note type="important">
     ///         At the time of writing, <see cref="Around"/> is not yet implemented into 
-    ///         <see cref="IMessageChannel.GetMessagesAsync"/>. Attempting to use the method with <see cref="Around"/>
-    ///         as its direction will throw a <see cref="System.NotImplementedException"/>.
+    ///         <see cref="IMessageChannel.GetMessagesAsync(int, CacheMode, RequestOptions)"/>.
+    ///         Attempting to use the method with <see cref="Around"/> will throw
+    ///         a <see cref="System.NotImplementedException"/>.
```

Comment:
Would this be a good fit for the `exception` tag? Given that it's not the intended design to throw `NotImplementedException` I'm not sure.